OpenRA.Mods.Common/Widgets/TextNotificationsDisplayWidget.cs

```diff
@@ -54,6 +62,8 @@ public override void Initialize(WidgetArgs args)
 			overflowDrawBounds = new Rectangle(RenderOrigin.X, RenderOrigin.Y, Bounds.Width, Bounds.Height);
 			overflowDrawBounds.Y += Bounds.Height - visibleChildrenHeight;
 			overflowDrawBounds.Height = visibleChildrenHeight;
+
+			durationTicks = DisplayDurationMs / world.Timestep;
```

Comment:
This doesn't adapt to changing the timestamp in replays.

Reply:
I gave this some thought and actually implemented scaling of the duration in replays. But I wonder - does the duration need to scale in replays at all? It's the moment a message appears on the screen that's important not whether it stays the same relative time on screen.